src/components/bulkConcepts/addBulkConcepts.jsx

```diff
@@ -138,6 +161,84 @@ Dictionary
               </label>
             </div>
             <br />
+
+
+            <div id="other-search">
+              <div className="form-check">
+                <input
+                  className="form-check-input"
+                  type="radio"
+                  name="exampleRadios"
+                  id="otherSourcesOption"
+                  onClick={this.otherSourceClick}
+                  value="option3"
+                />
+                <Downshift
+                  id="sourceSearchInput"
+                  onChange={selected => this.handleSourceSelect(selected)}
+                  itemToString={source => source && source.name}
+                >
+                  {({
+                    getInputProps,
+                    getItemProps,
+                    getMenuProps,
+                    highlightedIndex,
+                    isOpen,
+                    inputValue,
+                  }) => (
+                    <div>
+                      <div id="other-search">
+                          Other &nbsp;&nbsp;
+                        <form className="form-inline search-bar">
+                          <i className="fas fa-search" />
+                          { otherSelected && <input
+                            {
+                            ...getInputProps()
+                            }
+                            className="form-control search"
+                            id="sourceSearch"
+                            placeholder="search"
+                            aria-label="Search"
+                          />
+                          }
+                        </form>
+                      </div>
+                      <div className="search-ul">
+                        {isOpen ? (
+                          <ul {...getMenuProps()} className="search-ul">
+                            {
+                            conceptSources.filter(item => !inputValue.trim()
+                            || item.name.toLowerCase()
+                              .includes(inputValue.toLowerCase())).slice(1, 10)
+                              .map((item, index) => (
+                                <li
+                                  {...getItemProps({
+                                    key: item.id,
+                                    index,
+                                    item,
+                                    style: {
+                                      backgroundColor:
```

Comment:
Can't we do this from styling file?

Reply:
No, it is passed as props to downshift directly, and contains logic